app/services/cocktail_service.rb

```diff
@@ -0,0 +1,16 @@
+class CocktailService
+  def self.get_cocktail_details(cocktail_id)
+    response = Faraday.get('https://www.thecocktaildb.com/api/json/v1/1/lookup.php?') do |req|
+      req.params['i'] = cocktail_id
+      req.params['api_key'] = ENV['cocktail_key']
+    end
```

Comment:
I'd agree with @twarbelow that eventually we should add a method for independently creating the DB connection, but I don't think that's critical for this iteration, since we're only calling it the one time. Could be a future refactor imo

Reply:
Thoughts on that @twarbelow ?